github/client.go

```diff
@@ -152,6 +152,67 @@ func (c *Client) ReadReleaseByTag(ctx context.Context, tkn string, repoURL url.U
 	return model.ToSvcGithubRelease(rls)
 }
 
+// TODO once svcmodel.Release has git tag field, no gitTag parameter will be needed
+func (c *Client) UpdateRelease(ctx context.Context, tkn string, repo svcmodel.GithubRepo, rls svcmodel.Release, gitTag string) error {
```

Comment:
Once https://github.com/jan-zabloudil/release-manager/pull/116 is merged, I dont need to pass gitTag, because gitTag will be part of Release struct.